woodwork/column_schema.py

```diff
@@ -141,6 +146,15 @@ def is_boolean(self):
         ltype_class = type(self.logical_type)
         return ltype_class == Boolean or ltype_class == BooleanNullable
 
+    def is_nullable(self, series_type):
+        """Whether the logical type contains a nullable physical type.
+
+            Args:
+                series_type (Series): Type of series instance
+        """
+        dtype = self.logical_type._get_valid_dtype(series_type)
+        return dtype in _NULLABLE_PHYSICAL_TYPES
```

Comment:
> We can also add an `is_nullable` function on the ColumnSchema

@gsheni  do we want this to be a function of a series type?

Reply:
@rwedge @tamargrey @thehomebrewnerd @davesque can better answer this. I am not sure.

Reply:
Just glancing through this quickly, so I might be missing something, but generally we are expecting that a schema won't have any knowledge of the data to which it is attached (in some cases it won't have data - like in FT). So if we need to use the series type to determine if a column is nullable, I'm thinking this shouldn't be on the schema, but only on the accessor.

Reply:
If we're all on the same page here, I'll go ahead and not add  `is_nullable` on the ColumnSchema